canarieapi/schema.py

```diff
@@ -17,35 +18,34 @@
 
 
 def validate_config_schema(update_db: bool, run_jobs: bool = True) -> None:
-    # type: (bool) -> None
     config = APP.config
     logger: logging.Logger = APP.logger
 
     logger.info("Testing configuration...")
-    if config.get("PARSE_LOG"):
-        CONFIGURATION_SCHEMA["definitions"]["service_description_schema"]["required"].append("stats")
-        CONFIGURATION_SCHEMA["definitions"]["platform_description_schema"]["required"].append("stats")
+
+    configuration_schema = copy.deepcopy(CONFIGURATION_SCHEMA)
+    if config.get("PARSE_LOG", True):
+        configuration_schema["definitions"]["service_description_schema"]["required"].append("stats")
+        configuration_schema["definitions"]["platform_description_schema"]["required"].append("stats")
```

Comment:
Might be worth adding a `$comment` in the schema of `service_description_schema` and `platform_description_schema` to indicate this dynamic change according to `PARSE_LOG` value.